Gemfile

```diff
@@ -15,7 +18,15 @@ gem 'json-patch'
 
 gem 'savon-multipart'
 
+gem 'sidekiq'
+gem "kaminari"
```

Comment:
seems like there's a combo of single and double quotes in here -- any sense on if one or the other is used more in this project? if so, let's move toward whatever that is